jdrfandroidbleparser/src/test/java/org/ehealthinnovation/jdrfandroidbleparser/bgm/characteristic/compoundcharacteristic/GlucoseMeasurementContextCharacteristicTest.kt

```diff
@@ -1,13 +1,200 @@
 package org.ehealthinnovation.jdrfandroidbleparser.bgm.characteristic.compoundcharacteristic
 
 import org.ehealthinnovation.jdrfandroidbleparser.BaseTest
+import org.ehealthinnovation.jdrfandroidbleparser.encodedvalue.GattCharacteristic
+import org.ehealthinnovation.jdrfandroidbleparser.encodedvalue.Units
+import org.ehealthinnovation.jdrfandroidbleparser.encodedvalue.bgm.contextmeasurement.*
+import org.junit.Assert
+import org.junit.Before
 import org.junit.Test
 
-import org.junit.Assert.*
+class GlucoseMeasurementContextCharacteristicTest : BaseTest() {
 
-class GlucoseMeasurementContextCharacteristicTest: BaseTest() {
+    /*
+     * Test Packet 1
+     */
+    private val testPacket1: ByteArray = byteArrayOf(0x84.toByte(), 0x01.toByte(), 0x00.toByte(), 0xbb.toByte(), 0x51.toByte())
+    private val sequenceNumber1: Int = 1
+    private val extendedFlag1: Int = 187
```

Comment:
This can be written in hex code, which can be mapped directly to the binary packet without doing an conversion.

Reply:
I don't do a conversion. The flag is stored in the characteristic as an Int? value. So I compare the Int stored internally with this Int, also ensuring the signed/unsigned remains consistent.

Reply:
I mean, we can write 0xbb instead of 187 for readability sake.

Reply:
But if the value exceed a certain Hex value, then we have to considered signed vs unsigned, and would require explicit calls to toByte() to ensure consistency. As it stands, the characteristic class already parses, and stores it in the signed/unsigned desired final state. So I do the comparison against that.